README.md

```diff
@@ -1,70 +1,142 @@
+- Track JavaScript linter errors run:
 
-### Analyzing the Bundle Size
+npx eslint .
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
 
-### Making a Progressive Web App
+### Usage
+To run the project, follow these instructions:
+- After Cloning this repo to your local machine.
+- Open the `index.html` in your browser.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+### Run tests
+To run tests, run the following command:
+- Track HTML linter errors run:
 
-### Advanced Configuration
+npx hint .
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+- Track CSS linter errors run:
 
-### Deployment
+npx stylelint "**/*.{css,scss}"
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+- Track JavaScript linter errors run:
 
-### `npm run build` fails to minify
+npx eslint .
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+
+### Deployment <a name="deployment"></a>
+- Deployed with github pages
+- For more information about publishing sources, see "[About GitHub pages](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#publishing-sources-for-github-pages-sites)".
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :busts_in_silhouette: Authors <a name="authors"></a>
+ Emmanuel Kipngeno
+- GitHub: [@githubhandle](https://github.com/kkmanuu)
+- Twitter: [@twitterhandle](https://twitter.com/kkmanuu)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/emmanuel-kipngeno-879370242/)
+
+
+## :telescope: Future Features <a name="future-features"></a>
+
+-  Setting up User-Friendly List Display and Filtering
+-  Implementing API Integration and Data Display
+-  Creating Test Cases for Application Reliability
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :star:️ Show your support <a name="support"></a>
+If you like this project give start.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :handshake: Contributing <a name="contributing"></a>
+Contributions, issues [page issues](https://github.com/kkmanuu/math-magicians/issues), and feature requests are welcome!
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## :pray: Acknowledgments <a name="acknowledgements"></a>
+I would like to thank  Microverse for this opportunity.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Kindly add in your "Acknowledgements" section an acknowledgement that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Reply:
@LionRouge1 Done making the required changes.